src/components/SimpleFlowchart.vue

```diff
@@ -177,22 +175,23 @@ export default {
       // }
       // check if start node, then add margin top by 50px (manually)
       // let additionalHeight = 0
-      // if(isStart) {
+      // if(node.isStart) {
       //   additionalHeight += 50;
       // }
 
       if (type === 'right') {
-        if (buttonId) {
-          const buttonIndex = fromNode.buttons.findIndex((item) => item.id === buttonId);
+        if (buttonId && buttonId !== -1) {
+          const buttonIndex = node.buttons.findIndex((button) => button.id === buttonId);
 
-          return [x + labelWidth, y + labelHeight + 41 * (buttonIndex + 0.5 - fromNode.buttons.length)]
+          return [x + labelWidth, y + labelHeight + 41 * (buttonIndex + 0.5 - node.buttons.length)]
         } else {
-          if (this.draggingLink && this.draggingLink.buttonIndex) {
+          if (buttonId === -1 && this.draggingLink && this.draggingLink.buttonIndex) { // this line is important! -1 means the condition is in dragginglink
```

Comment:
this.draggingLink.buttonIndex with value of 0 will return false on this condition